jdk/make/closed/autoconf/custom-spec.gmk.in

```diff
@@ -72,3 +72,7 @@ OPENJ9_LIBS_SUBDIR      := @OPENJ9_LIBS_SUBDIR@
 
 # Windows
 MSVCP_DLL               := @MSVCP_DLL@
+
+ifeq ($(OPENJ9_BUILDSPEC), win_x86)
```

Comment:
Can we make this more generic? Probably just always set COPY_JVM_CFG_FILE := true would be fine for OpenJ9.

Reply:
Already removed the if statement.